guides/faq.rst

```diff
@@ -11,7 +11,7 @@ Tips for using ESPHome
 1. ESPHome supports (most of) `Home Assistant's YAML configuration directives
    <https://www.home-assistant.io/docs/configuration/splitting_configuration/>`__ like
    ``!include`` and ``!secret``. So you can store all your secret WiFi passwords and so on
-   in a file called ``secrets.yaml`` within the directory where the configuration file is.
+   in a file called ``secrets.yaml`` within the directory where the esphome configuration files are.
```

Comment:
Actually, the previous statement was correct.

> ``secrets.yaml`` within the directory where the configuration file is.

As they need to be next to each other, the configuration file that uses the secrets and the secrets file.

> within the directory where the esphome configuration files are.

This assumes there is only one directory, which doesn't have to be the case.